src/lib.js

```diff
@@ -253,11 +257,10 @@ export default class Migrator {
           migrationName = migrationToImport.slice(timestampSeparatorIndex + 1, migrationToImport.lastIndexOf('.'));
 
         this.log(`Adding migration ${filePath} into database from file system. State is ` + `DOWN`.red);
-        const createdMigration = MigrationModel.create({
+        return MigrationModel.create({
```

Comment:
As mentioned above, I tried with async, exactly as you write, it *does not work*:

```
SyntaxError: src/lib.js: await is a reserved word (260:33)
  258 | 
  259 |         this.log(`Adding migration ${filePath} into database from file system. State is ` + `DOWN`.red);
> 260 |         const createdMigration = await MigrationModel.create({
      |                                  ^
  261 |           name: migrationName,
  262 |           createdAt: timestamp
  263 |         });
```